resources/osl_mongodb.rb

```diff
@@ -0,0 +1,62 @@
+resource_name :osl_mongodb
+provides :osl_mongodb, platform_family: 'rhel'
+unified_mode true
+
+default_action :install
+
+property :version, String, name_property: true
+property :data_dir, String, default: '/var/lib/mongo/'
```

Comment:
@ArtofBugs is the trailing `/` required? I'd rather remove it if not.

Reply:
I don't think so, I will try removing it.

Reply:
@ArtofBugs don't forget to address this